tests/eclient/Makefile

```diff
@@ -49,6 +49,8 @@ test:
 build: setup
 
 setup:
+	$(LOCALBIN) utils template eden+ports.sh.tmpl > eden+ports.sh
```

Comment:
Hey @sadov @giggsoff @mydatascience -- can we please add an explicit note about `eden utils template` to our docs some place? It seems super useful and I actually didn't know it existed until now.

Reply:
Actually we have some description about `eden utils template` command and EDEN-specific Go template extension in [tests/README.md](https://github.com/lf-edge/eden/blob/master/tests/README.md#go-template-support), but yes -- may be it would be reasonable make it more informative and visible.